README.md

```diff
@@ -35,6 +35,7 @@ To further configure your plugin, use the following custom parameters in your `s
 | `forwarder`          | Setting this parameter subscribes the Lambda functions' CloudWatch log groups to the given Datadog forwarder Lambda function. Required when `enableDDTracing` is set to `true`.                                                                                                                                                                                                                 |
 | `enableTags`         | When set, automatically tag the Lambda functions with the `service` and `env` tags using the `service` and `stage` values from the serverless application definition. It does NOT override if a `service` or `env` tag already exists. Defaults to `true`.                                                                                                                                      |
 | `injectLogContext`         | When set, the lambda layer will automatically patch console.log with Datadog's tracing ids. Defaults to `true`.                                                                                                                                      |
+| `exclude`         | When set, this plugin ignore all specified functions. Use this parameter if you have any functions that should not include Datadog functionality. Defaults to `[]`.                                                                                                                                      |
```

Comment:
`this plugin ignore all` -> `this plugin will ignore all`

Also, I think it's worth specifying that this should be a comma-separated list of function names

Finally, is it normal that we expect the parameter to include [] brackets? Would it make more sense to just ask for a comma-separated string?

Reply:
I think it's easier to use a list/array, since that's supported natively by yaml, and we don't need to do any extra parsing ourselves. For instance:

```
exclude:
 - first-excluded-func
 - seconded-excluded-fun
```

and 

```
exclude: [first-excluded-func, second-excluded-func]
```

Are equivalent.

Reply:
Got it, but could we interpolate the string into the brackets? Rather than asking the user to supply the brackets?

If not, we should make it clear in the description that the parameter value must include the brackets.

Reply:
Oh, is this going to be something that the user types into the YML file themselves? In that case never mind. I think it's still worth mentioning it's a comma-separated list of function names.

Reply:
I updated the eample to use the regular yaml list syntax, rather than the bracket syntax.